spec/spiritual/spiritual-edb/arrays/edb.ArrayPopulator.spec.js

```diff
@@ -24,10 +24,9 @@ describe('edb.ArrayPopulator', function likethis() {
 	});
 
 	it('should fail on bad input', function() {
-		var persons; // eslint-disable-line no-unused-vars
 		var fail = false;
 		try {
-			persons = new PersonCollection([false]);
+			var persons = new PersonCollection([false]); // eslint-disable-line no-unused-vars
```

Comment:
do we need to assign this to a variable?

Reply:
If we remove this assign, eslint will start to complain `error : Do not use 'new' for side effects no-new`
Which rule you want to disable? :)

Reply:
Then left it as it is :D I forgot that also something that makes es-lint complain